examples/fips/README.md

```diff
@@ -0,0 +1,81 @@
+# FIPS Compliance
+
+## Communication between client and Kafka cluster
+
+### Install from prebuilt wheels
+
+If this client is installed using `pip`, it comes with librdkafka in which OpenSSL 3.0 is statically linked. This client can communicates with Kafka cluster with FIPS approved algorithms using FIPS provider provided by OpenSSL.
```

Comment:
Here, is the link to "client-configuration-to-enable-fips-provider" required? We need to still change the client configuration, right?

Or do we need to skip to another section instead?

Reply:
The `enabling FIPS provider` section is needed for this. I think I will make it more clear.